projects/plugins/jetpack/modules/subscriptions.php

```diff
@@ -1016,8 +1016,13 @@ public function add_subscribers_menu() {
 
 		$blog_id = Connection_Manager::get_site_id( true );
 
+		$source = 'jetpack-menu-calypso-subscribers';
+		if ( get_option( 'wpcom_admin_interface' ) === 'wp-admin' ) {
```

Comment:
Should we be checking for `get_option( 'wpcom_admin_interface' ) === 'wp-admin'` or `wpcom_is_nav_redesign_enabled()`? Jetpack Cloud is only enabled for the latter.

Or maybe we (I) made a mistake in D140513-code, that we actually shouldn't add the `wpcom_classic_early_release` check to Jetpack Cloud? (I think this makes more sense)

Reply:
> Should we be checking for get_option( 'wpcom_admin_interface' ) === 'wp-admin' or wpcom_is_nav_redesign_enabled()? Jetpack Cloud is only enabled for the latter.

I believe Jetpack Cloud is available to all Atomic sites via deep link. However, the code in D140513-code does affect how these sources redirect: fbhepr%2Skers%2Sjcpbz%2Sjc%2Qpbagrag%2Syvo%2Swrgcnpx%2Qerqverpgf%2S0%2Qybnq.cuc%3Se%3Q55o7rp6p%23317-og

In the case of this PR, Classic-only should be able to link to Jetpack Cloud "Subscribers" without issue, since it's not conditionally redirecting.

Is there another concern? 

> Or maybe we (I) made a mistake in D140513-code, that we actually shouldn't add the wpcom_classic_early_release check to Jetpack Cloud? (I think this makes more sense)

I think D140513-code is ok. Classic-only still has nav unification enabled, so pointing those redirect sources (linked above) for Classic to Jetpack Cloud means we probably need to update/remove some nav unification menu items in the Classic-only view. (Like how we need to remove the "Subscribers" menu item in the "Users" menu in this PR.

For this PR I bound the change to Classic, but we could just as easily limit it to Classic + `wpcom_classic_early_release`, or update it for all Atomic and Jetpack sites as @simison suggested above: https://github.com/Automattic/jetpack/pull/36510/files#r1535301835. 🤔

Reply:
> However, the code in [D140513-code](https://href.li/?https://code.a8c.com/D140513) does affect how these sources redirect:

Ahh, that's why! I wondered why redirects such as `calypso-backups` will redirect to wp.com for non-early-access sites. I thought it will the case for all Jetpack redirects, turned out there is a static list. 🤦 

But then I wonder why we would have different treatment for different sources e.g. this Subcscribers and Backup as I mentioned above 🤔 but maybe it's out of the scope of this PR